pkg/domain/events.go

```diff
@@ -77,22 +77,39 @@ func (e GlobalConfigDiffDeterminedEvent) Message() string {
 }
 
 type DoguConfigDiffDeterminedEvent struct {
-	CombinedDogusConfigDiffs map[common.SimpleDoguName]CombinedDoguConfigDiffs
+	DoguConfigDiffs          map[common.SimpleDoguName]DoguConfigDiffs
+	SensitiveDoguConfigDiffs map[common.SimpleDoguName]SensitiveDoguConfigDiffs
+}
+
+func NewDoguConfigDiffDeterminedEvent(
+	doguConfigDiffs map[common.SimpleDoguName]DoguConfigDiffs,
+	sensitiveDoguConfigDiffs map[common.SimpleDoguName]SensitiveDoguConfigDiffs) *DoguConfigDiffDeterminedEvent {
+	return &DoguConfigDiffDeterminedEvent{DoguConfigDiffs: doguConfigDiffs, SensitiveDoguConfigDiffs: sensitiveDoguConfigDiffs}
 }
```

Comment:
I think, I prefer the Intellij suggestion to put everything on seperate lines, because it makes the start of the function more clear to me:

```suggestion
func NewDoguConfigDiffDeterminedEvent(
	doguConfigDiffs map[common.SimpleDoguName]DoguConfigDiffs,
	sensitiveDoguConfigDiffs map[common.SimpleDoguName]SensitiveDoguConfigDiffs,
) *DoguConfigDiffDeterminedEvent {
	return &DoguConfigDiffDeterminedEvent{DoguConfigDiffs: doguConfigDiffs, SensitiveDoguConfigDiffs: sensitiveDoguConfigDiffs}
}
```